src/mesh.cpp

```diff
@@ -2309,10 +2321,20 @@ void MOABMesh::build_kdtree(const moab::Range& all_tets)
   all_tets_and_tris.merge(all_tris);
 
   // create a kd-tree instance
+  write_message("Building adaptive k-d tree for tet mesh...", 7);
   kdtree_ = make_unique<moab::AdaptiveKDTree>(mbi_.get());
 
-  // build the tree
-  rval = kdtree_->build_tree(all_tets_and_tris, &kdtree_root_);
+  // Determine what options to use
+  std::ostringstream options_stream;
+  if (options_.empty()) {
+    options_stream << "MAX_DEPTH=20;PLANE_SET=2;";
```

Comment:
I think the documentation should maybe tell users what the default values are in OpenMC.

Also: it's worth considering coding the default options in the python instead.

Reply:
I thought about that but then it would always show up in the XML file whenever an unstructured mesh was used, which seemed a little much.